### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by Jac-Zac** [pystardust/ytfzf#47] [Ideas]
Would you like to have a community discord server which might be good to work share ideas and solve problems faster ? I have made this to start and we can modify it and tailor it to our needs https://discord.gg/FNym2zpK

**Comment by westofer** [comment]
new link, old one expired https://discord.gg/kupWznHjRJ

**Reply by ethan-schoenhofen** [reply]
What about a Matrix server instead?
